build.sbt

```diff
@@ -12,3 +12,15 @@ daffodilFlatLayout := true
 daffodilVersion := daffodilPackageBinVersions.value.head
 
 daffodilPackageBinVersions := Seq("3.5.0", "3.8.0", "3.7.0")
+
+daffodilPackageBinInfos := Seq(
+  DaffodilBinInfo("fakeTDL.dfdl.xsd", Some("fakeTDL")) // schema for a single message (in src)
```

Comment:
fakeTDL.xsd needs to start with a `/`, that should allow it to generate a saved parser.